packages/itwinui-css/src/table/table.scss

```diff
@@ -154,11 +154,7 @@
 
     &:hover:not([aria-disabled='true']) {
       .iui-table-cell {
-        background: linear-gradient(
-          rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6)),
-          rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6))
-        ),
-          linear-gradient(var(--iui-color-background-1), var(--iui-color-background-1));
+        background-color: var(--iui-color-background-transparent-hover);
```

Comment:
I don't like this color change.

Reply:
This was done to make our hover states more consistent